open-source-software-peer-review/intro.md

```diff
@@ -1,13 +1,127 @@
 # How Peer Review Works
 
+```{tableofcontents}
+```
 
+## Why do we need pyOpenSci
 
+Discussion here on 
 
-```{tableofcontents}
+## Why submit your package to pyOpenSci for review?
+
+There is a lot to gain from submitting your package to pyOpenSci.
+But of course, before you submit, please be sure to read the policies 
+... to ensure pyOpenSci is the right fit for you. 
+
+First, and foremost, we hope you submit your package for review **because you value the feedback**.  We aim to provide useful feedback to package authors and for our review process to be open, non-adversarial, and focused on improving software quality.
+
+
+- Once your package is accepted, your package will receive **support from pyOpenSci members**. You'll retain ownership and control of your package. However, if you need help with ongoing maintenance issues, we will try to find people who can help. 
+  
+-  pyOpenSci will **promote your package** through our:
+   - [webpage](https://pyopensci.org/python-packages/), 
+   - [blog](https://wwwpyopensci.org/blog/), and 
+   - [social media account](https://twitter.com/pyopensci).
+
+-   pyOpenSci packages that are in scope for the [Journal of Open-Source Software](https://joss.theoj.org/) and add the necessary accompanying short `paper.md` file, can, at the discretion of JOSS editors, benefit from a fast-tracked review process. <LINK TO MORE ON THIS>
```

Comment:
This link needs to be added. where in our docs is it?
Also i don't see joss mentioned in the reviewer guide looking for it. that is where i'd expect it to be mentioned.